marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/artifacts/suiteTeardown.sjs

```diff
@@ -0,0 +1,5 @@
+const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
+const dataHub = DataHubSingleton.instance();
+
+dataHub.hubUtils.deleteDocument('/entities/TestEntity-NoMappingConfig', dataHub.config.STAGINGDATABASE);
```

Comment:
Don't need to do this either, just call hub-test:delete-artifacts($test:__CALLER_FILE__)